### ISSUE TITLE
Deploy Chaincode using hl-composer

### ISSUE THREAD

**Issue by amritg** [hyperledger-labs/nephos#32]
I am learning Hyperledger on my own and pretty new to Kubernetes (which is the problem). Just started to try Nephos, I must say Nephos abstracts away a lot of complex parts. I followed steps in documentation to run test example in my minikube and it works well. 
Now, I need to deploy .bna file to the cluster developed using Composer. Then, I also find the presence of 'hl-composer' chart which should be able to install chaincodes using Composer on the K8s cluster but I couldn't find any example on how can I do so. I got stuck now. So, It will nice, for the beginner like me, to have a few more instructions in documentation to be able to install chaincode using hl-composer and run rest-server, and playground in minikube.

Thanks in advance for any possible directions!

**Comment by alexvicegrab** [comment]
Hi Amrit,

You can attempt running the QA example (which requires a running CA), and run the `deploy.py` script with the `deploy` command (or `fabric` followed by `composer`).

We are happy to receive PRs improving the documentation.

**Comment by amritg** [comment]
Hi @alexvicegrab ,
Thanks for the quick reply. Can I run CA locally on minikube in order to run QA example? In documentation, Its says 'Given that we may wish to test locally on Minikube, we will need to use a local ingress controller and ignore cert-manager in favor of self-cooked SSL certificates.'

**Comment by alexvicegrab** [comment]
Yes, you can, but your CA will point to a local "pretend" address, which will need to be added to `/etc/hosts` as described [here](https://github.com/hyperledger-labs/nephos#minikube)

The QA example includes a CA, as you can see if you run it, and in the `nephos_config.yaml`
